ReactiveCocoa/NSObject+Intercepting.swift

```diff
@@ -155,8 +155,18 @@ private func enableMessageForwarding(_ realClass: AnyClass, _ selectorCache: Sel
 			}
 		}
 
-		let method = class_getInstanceMethod(perceivedClass, selector)!
```

Comment:
We currently assume that, if the intercepted class has declared its support of a selector via `-respondsToSelector:`, there _must_ be an implementation registered with the Objective-C runtime.

However in practice this is not often the case. For example, we have encountered a crash when intercepting `JSQMessageCollectionCellView` from the JSQMessageViewController framework, whose message forwarding based handling of menu actions clashes with the said assumption.